public/static/locales/en/apps.json

```diff
@@ -17,6 +17,8 @@
     "avuFetchError": "Unable to get app metadata. Please try again.",
     "authenticateBtnText": "Authenticate",
     "beta": "Beta Tag Applied",
+    "blessed": "Blessed By CyVerse",
+    "blessedToolTip": "This app is integrated, verified thats it is working and kept up to date by CyVerse staff.",
```

Comment:
Maybe `This app is integrated, verified, and kept up to date by CyVerse staff.` to be a little shorter